src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/ActionModalForm.tsx

```diff
@@ -222,49 +219,43 @@ class ActionModalForm extends Component<Props, State> {
 
   render() {
     return (
-      <Modal
-        animation={false}
-        show={this.props.show}
-        onHide={this.onClose}
-        container={document.getElementById('app')}
-        onKeyDown={this.handleKeyDown}
-        backdrop={'static'}
+      <BaseDialog
+        title={this.state.isEdit ? lang.tr('studio.flow.node.editAction') : lang.tr('studio.flow.node.addAction')}
+        isOpen={this.props.show}
+        onClose={this.onClose}
```

Comment:
Don't forget that BaseDialog already has the onSubmit property, which will wrap the content in a form automatically

Reply:
thanks for the pointer, fixed it and I came across an issue when there was a form and scrollbar at the same time